canvas_modules/common-canvas/src/common-canvas/common-canvas.jsx

```diff
@@ -405,7 +405,11 @@ class CommonCanvas extends React.Component {
 				this.state.rightFlyoutContent !== null &&
 				this.props.showRightFlyout) {
 			rightFlyout = (
-				<div className={"right-flyout-panel"}>
+				<div
```

Comment:
Shouldn't this go into common-properties instead of common-canvas?

Reply:
In common-properties.jsx, I have to add landmark role at multiple places for all conditions at line numbers - [64](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L64), [135](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L135), [151](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L151), [170](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L170), [189](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L189).

To avoid this duplication, I declared landmark role in common-canvas.

Reply:
Wouldn't a consumer that only users common-properties need these set in common-properties?